apps/dashboard/config/routes.rb

```diff
@@ -7,6 +7,8 @@
     resources :projects do
       root 'projects#index'
       get '/jobs/:cluster/:jobid' => 'projects#job_details', :defaults => { :format => 'turbo_stream' }, :as => 'job_details'
+      delete '/jobs/:cluster/:jobid' => 'projects#delete_job', :as => 'delete_job'
+      patch '/jobs/:cluster/:jobid/stop' => 'projects#stop_job', :as => 'stop_job'
```

Comment:
Interesting choice of `PATCH` here. Not that it's wrong, I guess I have to confer with the RFC.